app/services/twilio_api_service.rb

```diff
@@ -71,12 +68,16 @@ def test_client
   end
 
   def send_sms(recipient_number:, message:, callback_url: nil, context: {})
+    self.communication_type = "sms"
```

Comment:
This is hacky, but until we refactor how the Worker works with this service it will be fine.

Reply:
Any reason you prefer having he accessor and setting via self rather than using a reader and setting via instance variable?

Reply:
I think it is a bit cleaner and will make things a bit easier to refactor later.  Ideally we wouldn't have to set communication_type here at all, but instead could do it earlier in the initialize...but that will require some more refactoring.